dash/App/Tokens/AddToken/index.js

```diff
@@ -5,66 +5,28 @@ import chainMeta from '../../../../resources/chainMeta'
 import link from '../../../../resources/link'
 import svg from '../../../../resources/svg'
 
-class AddToken extends Component {
+class AddTokenChainScreen extends Component {
   constructor (props, context) {
     super(props, context)
-
-    this.nameDefault = 'Token Name'
-    this.symbolDefault = 'SYMBOL'
-    // this.chainIdDefault = 1
-    this.decimalsDefault = '?'
-    this.addressDefault = 'Contract Address'
-    this.logoURIDefault = 'Logo URI'
-
-    this.req = props.req || {}
-    this.token = this.req.token || {}
-    this.activeChains = props.activeChains
-
-    const chainId = parseInt(this.token.chainId)
-    const decimals = parseInt(this.token.decimals)
-
-    this.state = {
-      name: this.token.name || this.nameDefault,
-      symbol: (this.token.symbol || '').toUpperCase() || this.symbolDefault,
-      _chainId: Number.isInteger(chainId) && chainId,
-      chainId: Number.isInteger(chainId) && chainId,
-      inputAddress: (this.token.address || '').toLowerCase(),
-      address: (this.token.address || '').toLowerCase(),
-      decimals: (Number.isInteger(decimals) && decimals) || this.decimalsDefault,
-      logoURI: this.token.logoURI || this.logoURIDefault
-    }
-  }
-
-  async updateTokenData (contractAddress, chainId) {
-    const { name, symbol, decimals } = await link.invoke('tray:getTokenDetails', contractAddress, chainId)
-    this.setState({
-      name: name || this.nameDefault,
-      symbol: symbol || this.symbolDefault,
-      decimals: decimals || this.decimalsDefault
-    })
-  }
-
-  isConnectedChain () {
-    const chain = this.activeChains.find(({ id }) => id === this.state.chainId)
-    return chain.connection.primary.connected || chain.connection.secondary.connected
+    this.state = {}
   }
-
-  isDefault (statePropName) {
-    if (this.state[statePropName] === undefined) {
-      return false
-    }
-    return this.state[statePropName] === this[`${statePropName}Default`]
-  }
-
-  updateOriginChain () {
-    // const origin = this.store('main.origins', this.props.originId)
-    return (
+  
+  render () {
+    const { store } = this.props
+    const { chainId } = this.state
+    return <div className='newTokenView cardShow'>
+      <div className='newTokenChainSelectTitle'>
+        {'What chain is this token on?'}
+      </div>
+      <div className='newTokenChainSelectChain'>
       <div className='originSwapChainList'>
-        {Object.keys(this.store('main.networks.ethereum'))
-          .filter(id => this.store('main.networks.ethereum', id, 'on'))
+        {Object.keys(store('main.networks.ethereum'))
+          .filter(id => store('main.networks.ethereum', id, 'on'))
           .map(id => {
-            const hexId = '0x' + parseInt(id).toString('16')
-            const selected = this.state._chainId === parseInt(id)
+            const intId = parseInt(id)
+            const hexId = '0x' + intId.toString('16')
```

Comment:
the parameter to `toString()` is supposed to be a number. probably better to just use a utility `toHex` function here